templates/scrollPrompt.jsx

```diff
@@ -1,32 +1,31 @@
 import React from 'react';
-import { classes, compile } from 'core/js/reactHelpers';
+import { classes, compile, templates } from 'core/js/reactHelpers';
 
 export default function ScrollPrompt (props) {
   const {
-    _scrollPrompt,
-    onScrollPromptClick
+    _scrollPrompt
   } = props;
 
+  const buttonPosition = _scrollPrompt._buttonPosition;
```

Comment:
Provide a sensible default if `_buttonPosition` is not set.
```suggestion
  const buttonPosition = _scrollPrompt._buttonPosition || 'left';
```